src/store/annotations/reducer.ts

```diff
@@ -38,7 +38,25 @@ const annotationsById = createReducer<AnnotationsState['byId']>({}, builder =>
         })
         .addCase(fetchAnnotationsAction.fulfilled, (state, { payload }) => {
             payload.entries.forEach(annotation => {
-                state[annotation.id] = addClientIds(annotation);
+                let newAnnotation = annotation;
```

Comment:
We should try to keep this reducer callback as simple as possible. Can we add a formatting function that calls out to other type-specific formatting functions?

```
function format(annotation: Annotation): Annotation {
    let result = annotation;

    if (isDrawing(annotation) {
       result = formatDrawing(annotation);
    }

    return result;
}

state[annotatation.id] = format(annotation);
```